app/javascript/packs/initializers/initializeNotifications.js

```diff
@@ -198,3 +191,13 @@ function initLoadMoreButton() {
     button.addEventListener('click', initPagination);
   }
 }
+
+export function initializeNotifications() {
+  fetchNotificationsCount();
```

Comment:
I'm wondering if some of this code should be covered by cypress tests, this function almost reads like a user experience from start to finish hence it sounds a little interaction heavy and may possibly be a good candidate for an end to end test if it isn't covered already. What are your thoughts @lboogie04 ?